src/main/java/in/org/projecteka/hiu/HiuConfiguration.java

```diff
@@ -137,7 +137,8 @@ public ConsentService consentService(
             CentralRegistry centralRegistry,
             HealthInformationPublisher healthInformationPublisher,
             ConceptValidator validator,
-            GatewayServiceClient gatewayServiceClient) {
+            GatewayServiceClient gatewayServiceClient,
+            GatewayServiceProperties gatewayServiceProperties) {
```

Comment:
Does ConsentService need the gatewayServiceProperties? If not, then it should be injected to the GatewayServiceClient, right?

Reply:
We are needing this because we are using this in Granted Consent Task and we don't have any bean definition for that. So that's why currently it is being passed from here. In future when we create beans for each of the Consent Task then it won't no longer be here.

Reply:
ok